extensions/email/src/test/java/org/mule/extension/email/retriever/IMAPNegativeConnectivityTestCase.java

```diff
@@ -82,6 +83,7 @@ public void configCannotReach() {
     connectivityUtils.assertFailedConnection("configCannotReach", exceptionMatcher, is(errorType(UNKNOWN_HOST)));
   }
 
+  @Ignore
```

Comment:
Add a reference to an issue about fixing it. Otherwise this will be ignored for ever and you might as well delete it.